react/features/recording/components/LiveStream/AbstractLiveStreamButton.ts

```diff
@@ -131,8 +131,9 @@ export function _mapStateToProps(state: IReduxState, ownProps: IProps) {
         const isModerator = isLocalParticipantModerator(state);
         const liveStreaming = getLiveStreaming(state);
 
-        visible = isModerator && liveStreaming.enabled;
-        visible = isJwtFeatureEnabled(state, 'livestreaming', visible);
+        visible = isModerator
+            ? isJwtFeatureEnabled(state, 'livestreaming', liveStreaming.enabled)
```

Comment:
But this means that if there is a jwt with the feature enabled, but the config has livestreaming disabled we will still show the button.

Reply:
Good point!

Reply:
this was existing logic - if we want the config to trump the jwt feature, I can change.

Reply:
oh there are more params no just the last one